src/config.js

```diff
@@ -22,6 +22,7 @@ const defaultConfig = {
   tolerance: 0,
   input: {
     coverageSummaryPath: "coverage/coverage-summary.json",
+    coverageHtmlPath: "coverage/html-report",
```

Comment:
This section is for "inputs", a config that has to do with controlling where a generated report is "output" to should go in either output, or possibly in a new `report` key that has both the report type and it's configuration defined within it, I've seen the structure before in the jest config I think, offhand something like:

```
[ {"json-summary": {}}, {"html": {"outputPath": "/html"}]
```
```